pkg/util/namespace.go

```diff
@@ -72,3 +73,28 @@ func EnsureNamespaceExist(client kubeclient.Interface, namespace string, dryRun
 
 	return createdObj, nil
 }
+
+// CreateOrUpdateNamespace creates a Namespaces if the target resource doesn't exist. If the resource exists already, this function will update the resource instead.
+func CreateOrUpdateNamespace(client kubeclient.Interface, ns *corev1.Namespace) error {
```

Comment:
Suggest to move this function to `pkg/karmadactl/cmdinit/utils` for the moment.

Reply:
Ok,I will do it.